apps/barista-design-system/src/components/ba-decision-graph/ba-decision-graph-start-node/ba-decision-graph-start-node.html

```diff
@@ -3,14 +3,14 @@
     class="ba-uxd-startnode-wrapper"
     *ngFor="let startnode of decisionGraphStartnodes, let i = index"
   >
-    <button
+    <div
```

Comment:
Haven't found a way to remove `vertical-align: center` default style of the text in a button in the startnode template. The reason is I'm binding innerHTML to render SafeHtml and therefore no way of setting this element to a button without having that annoying centering. Any suggestions would be appreciated! :)